dbgpt/rag/transformer/graph_extractor.py

```diff
@@ -65,7 +65,7 @@ def _parse_response(self, text: str, limit: Optional[int] = None) -> List[Graph]
                     match = re.match(r"\((.*?)#(.*?)\)", line)
                     if match:
                         name, summary = [part.strip() for part in match.groups()]
-                        graph.upsert_vertex(Vertex(name, description=summary))
+                        graph.upsert_vertex(Vertex(name, description=summary, vertex_type='entity'))
```

Comment:
This is incorrect. The `vertex_type` and `edge_type` should be determined by `TuGraphStore` itself when inserting.